frontend/src/components/regions/UserActions/ActivityLog.jsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import TodayIcon from "@material-ui/icons/Today";
+import Grid from "@material-ui/core/Grid";
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles((theme) => ({
+  containerStyles: {
+    width: "50%",
+    margin: 5,
+    border: "1px black solid",
+  },
+
+  fillerStyles: {
+    height: "100%",
+    textAlign: "right",
+    display: "flex",
+    flexDirection: "row",
+  },
+  labelStyles: {
+    width: 20,
+    color: "black",
+  },
+  content: {
+    width: "90%",
+    ["@media (max-width:620px)"]: {
+      fontSize: "1rem",
+    },
+  },
+  calendarIcon: {
+    margin: theme.spacing(1),
+  },
+  dateTypography: {
+    marginTop: theme.spacing(2),
+    ["@media (max-width:620px)"]: {
+      fontSize: "1rem",
+    },
+  },
+}));
+
+export default ({ eventList, sortedTimestampArray }) => {
```

Comment:
another styleguide relate thing I missed is this functional component needs a to have a name